docs/source/tutorial.rst

```diff
@@ -279,19 +279,19 @@ Now, lets suppose you want to perform additional analyses based on the 16S rDNA.
 The :code:`bbduk` command can pick the 16S rDNA from your metagenomic samples through
 `BBDuk <https://jgi.doe.gov/data-and-tools/bbtools/bb-tools-user-guide/bbduk-guide/>`_.
 But first, you will need to download the 16S rDNA sequences from the database of
-your choice. We recommend to download the 16S rDNA sequences from the up-to-date
-`SILVA_16S database <https://www.arb-silva.de/>`_ and store it in a directory
-(:file:`SILVA_16S/`). In this case, we will use our custom 16S rDNA database located here:
-`custom 16S_SILVA_db <https://drive.google.com/file/d/1sK03oL5b3dvsWW_CKDhaZEfmfQ9c4tc4/view?usp=sharing>`_
+your choice. In this case, we will use our custom 16S rDNA database of the
+phylum `Firmicutes <https://drive.google.com/file/d/1dOIgupiE-xpORIR-7jxaTMI63NXQBvdH/view?usp=sharing>`_. Go ahead and run :code:`bbduk` command like so:
```

Comment:
I see everything fine, however I have some suggestions to improve this section:

- Extend the link to the Firmicutes database, something like `` `16S rDNA database of the
phylum Firmicutes <URL>`_``.
- Split long lines, 
  ```
  phylum `Firmicutes <https://drive.google.com/file/d/1dOIgupiE-xpORIR-7jxaTMI63NXQBvdH/view?usp=sharing>`_. Go ahead and run :code:`bbduk` command like so:
  ```
  Could be written like this:
  ```
  phylum `Firmicutes <https://drive.google.com/file/d/1dOIgupiE-xpORIR-7jxaTMI63NXQBvdH/view?usp=sharing>`_. 
  Go ahead and run :code:`bbduk` command like so:
  ```
- We should consider sharing files using something different to Drive because this method does not allow to download the file directly from the command line, and sometimes this is necessary, for example when working in a cluster.
- Maybe could be useful to tell the user where the file should be stored, as the file could be saved automatically in the `Downloads/` folder and the user could run the command without placing the file in the working folder.

Reply:
Hi Juan,

I agree with you. So if I understood you correctly, we can create a new directory called `Downloads` . So, all the required files for the tutorial can be used from that `dir` without downloading them from Drive. But, where do you think will be better to create this `dir`? in the `metabiome ` directory  or inside the `/docs/source/_static` directory? or you have already created it ?

I am gonna apply your suggestions in the meantime!! 
:safety_vest:

Reply:
Hi Cristian! No, I am not sure yet if its good idea to create a directory in the repository to store those files, but it can be an option. However we could consider another solutions such as storing the files in Zenodo and providing the download link in the tutorial. I would like to discuss this subject patiently, so I think we can continue with Drive meanwhile and change it later.

Thanks!